config/clusters/2i2c/jackeddy.values.yaml

```
@@ -0,0 +1,181 @@
+basehub:
+  userServiceAccount:
+    annotations:
+      iam.gke.io/gcp-service-account: pilot-hubs-jackeddy@two-eye-two-see.iam.gserviceaccount.com
+  jupyterhub:
+    # https://infrastructure.2i2c.org/howto/features/ephemeral/#pre-pulled-images
+    prePuller:
+      continuous:
+        enabled: true
+    ingress:
+      hosts:
+        - jackeddy.2i2c.cloud
+      tls:
+        - secretName: https-auto-tls
+          hosts:
+            - jackeddy.2i2c.cloud
+    custom:
+      2i2c:
+        add_staff_user_ids_to_admin_users: true
+        add_staff_user_ids_of_type: "github"
+      homepage:
+        templateVars:
+          org:
+            name: Jack Eddy Symposium
+            url: https://cpaess.ucar.edu/meetings/4th-eddy-cross-disciplinary-symposium
+            logo_url: https://cpaess.ucar.edu/sites/default/files/styles/extra_large/public/2023-08/EddySymposium-900x400.jpg?itok=8qG7Dqi3
+          designed_by:
+            name: 2i2c
+            url: https://2i2c.org
+          operated_by:
+            name: 2i2c
+            url: https://2i2c.org
+          funded_by:
+            name: ""
+            url: ""
+            # FIXME: document this feature https://github.com/2i2c-org/default-hub-homepage/pull/25
+            custom_html: <a href="https://science.nasa.gov/heliophysics/programs/living-with-a-star/">NASA's Living with a Star program</a> and <a href="https://cpaess.ucar.edu/">UCAR/CPAESS</a>
```

```suggestion
            custom_html: <a href="https://science.nasa.gov/heliophysics/programs/living-with-a-star/">NASA's Living with a Star program</a> and <a href="https://cpaess.ucar.edu/">UCAR/CPAESS</a>
```

I'd like to avoid a fixme note about something that is general to all hubs - specifically in one hub, so I opened https://github.com/2i2c-org/infrastructure/issues/3259

---

Good point! Thank you!